calcom/cal.com
# chore: reorganize event type files

## What does this PR do?

In preparation for EventTypeAtom, reorganized event-type related files



https://github.com/user-attachments/assets/9fcc86e1-e498-442b-a7a7-c58f4a4dafa2



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- Navigate to Event-Type update form
- Check all event-type tabs
